docs/ManualDeploymentGuideScalarDBClusterOnEKS.md

```diff
@@ -0,0 +1,59 @@
+# Deploy ScalarDB Cluster on Amazon Elastic Kubernetes Service (EKS)
+
+This guide explains how to deploy ScalarDB Cluster on Amazon Elastic Kubernetes Service (EKS).
+
+In this guide, you will create one of the following two environments in your AWS environment. The difference between the two environments is which [client modes](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/developer-guide-for-scalardb-cluster-with-java-api.md#client-modes) you use:
```

Comment:
There are two client modes in ScalarDB Cluster. So, there are two patterns for deployment on the Kubernetes environment.